tasks/gulp/asset-version.js

```diff
@@ -1,35 +1,49 @@
 'use strict'
 
 const configPaths = require('../../config/paths.json')
-const gulp = require('gulp')
 const fs = require('fs')
+const path = require('path')
 const taskArguments = require('./task-arguments')
-const gulpif = require('gulp-if')
-const rename = require('gulp-rename')
-const del = require('del')
-const vinylPaths = require('vinyl-paths')
 
 // check for the flag passed by the task
-
 const isDist = taskArguments.destination === 'dist' || false
 
 // Update assets' versions ----------
 // Add all.package.json version
```

Comment:
Is this still accurate? We're writing to `VERSION.txt`

Might also be nice to be more accurate with the description here, something like "append new asset version to file names".

Reply:
I think it's accurate, just poorly explained? 

I interpreted it as it adding the version *from* `package.json`. We should probably rewrite it so that it's obvious, though.

Reply:
Ah, right - I read it as "Add the version to package.json" at a glance 🤦 , so count me as the reason for spelling it out!